publisher-commons/src/main/java/nl/idgis/publisher/database/JdbcDatabase.java

```diff
@@ -108,7 +107,7 @@ public void postStop() throws Exception {
 		connectionPool.close();		
 	}
 	
-	protected abstract Props createTransaction(Connection connection);
+	protected abstract Props createTransaction(Config config, Connection connection);

```

Comment:
Toevoegen van 'config' parameter is feitelijk overbodig: subclasses van JdbcDatabase kunnen direct gebruik maken van het protected attribuut JdbcDatabase.config.